README.md

```diff
@@ -67,6 +67,18 @@ scraper to test out config changes.
   * Generate documentation for all the modules in your PDK (where possible) and
     put in a folder inside your version docs
 
+## Generating the Admin API Documentation
+
+- Have a local clone of Kong
+- Install resty-cli (comes with Kong)
```

Comment:
this is technically correct, but it can be confusing. I'd just go with: "make sure that `resty` is in your $PATH (in other words, have a locally installed copy of Kong)"